File: app/src/main/java/com/letianpai/robot/wificonnet/parser/BindStatusInfo.java
package com.letianpai.robot.wificonnet.parser;

public class BindStatusInfo {

    private int code;
    private String msg;
    private BindData data;

    public int getCode() {
        return code;
    }

    public void setCode(int code) {
        this.code = code;
    }

    public String getMsg() {
        return msg;
    }

    public void setMsg(String msg) {
        this.msg = msg;
    }

    public BindData getData() {
        return data;
    }

    public void setData(BindData data) {
        this.data = data;
    }

    @Override
    public String toString() {
        return "{" +
                "code=" + code +
                ", msg='" + msg + '\'' +
                ", data=" + data +
                '}';
    }
}

File: app/src/main/java/com/letianpai/robot/wificonnet/parser/BindStatusInfo.kt
package com.letianpai.robot.wificonnet.parser

class BindStatusInfo {
    var code: Int = 0
    var msg: String? = null
    var data: BindData? = null

    override fun toString(): String {
        return "{" +
                "code=" + code +
                ", msg='" + msg + '\'' +
                ", data=" + data +
                '}'
    }
}
